.github/workflows/nightly-main-latest.yaml

```diff
@@ -0,0 +1,12 @@
+name: Nightly CLI test for main
+
+on:
+  schedule:
+    - cron: '08 1,13 * * *'
+
+jobs:
+  test-suite:
+    uses: aufi/kantra-cli-tests/.github/workflows/test-suite.yaml@basic-workflows # Replace with konveyor-ecosystem (cannot do checkout before for different OSs)
```

Comment:
Will replace `aufi` fork in followup PRs, this is actually faster to execute tests with modified actions, also the workflow would fail immediately on not existing actions since it is not yet merged in main branch.